.buildkite/hooks/post-command

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# This script is executed by Buildkite on the host machine.
+# In contrast, our build jobs are run in Docker containers.
+# This means that even though our build jobs write to
+# `/artifact-mount`, the directory on the host machine is
+# actually `/tmp/artifacts`.
+# Here, we cat all text files in artifact-mount/test-summaries.
+# These files contain a condensed summary of all failing pytest
+# tests to make it easy for users to see which tests are failing.
+# Because we print them here, we don't need to persist them as
+# artifacts, so we remove the directory afterwards.
+set -e
```

Comment:
Let's only use `set -e` here, please (not `set -x`), as this is about readable output, and `set -x` would defeat that purpose

Reply:
can you leave a comment in the script? so we can prevent adding this in the future